docs/superpowers/plans/2026-08-08-scpdsi-pr2-self-calibration-stats.md

```
@@ -0,0 +1,1008 @@
+```
+
+- [ ] **Step 2: Run test to verify it passes**
+
+Run: `uv run pytest tests/test_self_calibration.py -v`
+Expected: PASS (32 tests). If `test_public_surface_is_exported` fails, an earlier task's `__all__` addition was missed — reconcile `__all__` against the File Structure section's list rather than editing the test.
+
+- [ ] **Step 3: Run the full verification sweep**
+
+```bash
+uv run pytest
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)